walkthroughs/week-2-web-development/examples/stanley/src/main/webapp/images.html

```diff
@@ -9,21 +9,51 @@
     <div id="content">
       <h1>Laura's Image Gallery</h1>
       <div class="image-thumbnails">
-        <a href="images-me/laura-1.jpg"><img src="images-me/laura-1.jpg"/></a>
-        <a href="images-me/laura-2.jpg"><img src="images-me/laura-2.jpg"/></a>
-        <a href="images-me/laura-3.jpg"><img src="images-me/laura-3.jpg"/></a>
-        <a href="images-me/laura-4.jpg"><img src="images-me/laura-4.jpg"/></a>
-        <a href="images-me/laura-5.jpg"><img src="images-me/laura-5.jpg"/></a>
-        <a href="images-me/laura-6.jpg"><img src="images-me/laura-6.jpg"/></a>
-        <a href="images-me/laura-7.jpg"><img src="images-me/laura-7.jpg"/></a>
-        <a href="images-me/laura-8.jpg"><img src="images-me/laura-8.jpg"/></a>
-        <a href="images-me/laura-9.jpg"><img src="images-me/laura-9.jpg"/></a>
-        <a href="images-me/laura-10.jpg"><img src="images-me/laura-10.jpg"/></a>
-        <a href="images-me/laura-11.jpg"><img src="images-me/laura-11.jpg"/></a>
-        <a href="images-me/laura-12.jpg"><img src="images-me/laura-12.jpg"/></a>
-        <a href="images-me/laura-13.jpg"><img src="images-me/laura-13.jpg"/></a>
-        <a href="images-me/laura-14.jpg"><img src="images-me/laura-14.jpg"/></a>
-	<a href="images-me/laura-15.jpg"><img src="images-me/laura-15.jpg"/></a>
+        <a href="images-me/laura-1.jpg">
+            <img src="images-me/laura-1.jpg"/>
+            <div class="caption">Monserrat, Spain</div></a>
+        <a href="images-me/laura-2.jpg">
+            <img src="images-me/laura-2.jpg"/>
+            <div class="caption">Tibidabo, Spain</div></a>
+        <a href="images-me/laura-3.jpg">
+            <img src="images-me/laura-3.jpg"/>
+            <div class="caption">The Bunker, Spain</div></a>
+        <a href="images-me/laura-4.jpg">
+            <img src="images-me/laura-4.jpg"/>
+            <div class="caption">Parc Guell, Spain</div></a>
+        <a href="images-me/laura-5.jpg">
+        <img src="images-me/laura-5.jpg"/>
+        <div class="caption">Turd Salt Mine, Romaina</div></a>
```

Comment:
Your indenting here is a bit inconsistent, would be nice to keep it the same throughout.

Reply:
You are completely right, did not realize will fix now!